flow/project.py

```diff
@@ -2077,75 +2156,104 @@ def _get_operations_status(self, jobs, cached_status):
         starting_dict = functools.partial(dict, scheduler_status=JobStatus.unknown)
         status_dict = defaultdict(starting_dict)
 
-        groups = [self._groups[name] for name in self.operations]
-        for group in groups:
-            if get_aggregate_id(jobs) in self._group_to_aggregator[group]:
-                completed = group._complete(jobs)
-                eligible = not completed and group._eligible(jobs)
-                scheduler_status = cached_status.get(
-                    group._generate_id(jobs), JobStatus.unknown
-                )
-                for operation in group.operations:
-                    if scheduler_status >= status_dict[operation]["scheduler_status"]:
-                        status_dict[operation] = {
-                            "scheduler_status": scheduler_status,
-                            "eligible": eligible,
-                            "completed": completed,
-                        }
+        single_operation_groups = {self._groups[name] for name in self.operations}
```

Comment:
Maybe add a TODO indicating that we're just not showing any meaningful status for groups at this stage.